application/src/main/kotlin/com/studentcenter/weave/application/meeting/service/domain/impl/MeetingTeamDomainServiceImpl.kt

```diff
@@ -61,6 +68,31 @@ class MeetingTeamDomainServiceImpl(
         }
     }
 
+    override fun updateById(
+        id: UUID,
+        location: Location?,
+        memberCount: Int?,
+        teamIntroduce: TeamIntroduce?,
+    ): MeetingTeam {
+        if (memberCount != null) {
+            checkMemberCountUpdatable(id, memberCount)
+        }
```

Comment:
p4) kotlin 스럽게 `?.let` 쓰기
```suggestion
        memberCount?.let { checkMemberCountUpdatable(id, it) }
```

Reply:
이거는 if쓸까 let쓸까 고민하긴했는데, 명시적이지 않은거 같아 if쓰긴했는데요, 동일한 의견이긴 해서 바꿀게요 👍🏼